### PR TITLE
Implement GetFollowupMessage & GetWebhookMessage

### PR DESCRIPTION
**Pull request by Lulalaby**
Implementation of GetFollowupMessage & GetWebhookMessage

See https://github.com/discord/discord-api-docs/pull/3478

### CODE REVIEW

File: DSharpPlus/Net/Rest/DiscordApiClient.cs [hunk 1]
```diff
@@ -2118,6 +2118,22 @@ internal async Task<DiscordWebhook> GetWebhookWithTokenAsync(ulong webhook_id, s
 
             return ret;
         }
+        
+        internal async Task<DiscordMessage> GetWebhookMessageAsync(ulong webhook_id, string webhook_token, string message_id)
```
Comment (IDoEverything) [inline DSharpPlus/Net/Rest/DiscordApiClient.cs:2122]: Why does the message id take a string?

Comment (Lulalaby) [reply DSharpPlus/Net/Rest/DiscordApiClient.cs:2122]: uhm idk, seemed right for me

Comment (Lulalaby) [reply DSharpPlus/Net/Rest/DiscordApiClient.cs]: fixed

**Comment by Lulalaby** [comment]
mhh appveyor does not test anymore..

**Comment by Lulalaby** [comment]
@IDoEverything could you take a look what appveyor has?

**Comment by Lulalaby** [comment]
Should be good now

**Comment by Lulalaby** [comment]
pr is merged in discord @Neuheit

**Pull request merged**